pkg/runtime/generate_test.go

```diff
@@ -106,7 +104,7 @@ CMD ["node", "functions/list.js"]`,
 		},
 		{
 			name:     "java",
-			f:        &stack.Function{Handler: "test.java", ComputeUnit: stack.ComputeUnit{Context: "testdata"}},
+			handler:  "testdata/test.java",
```

Comment:
Just want to make sure with this one, handler entrypoint for java at the moment are always jars (basically just fatjars, that will probably be pulled from /target/ or /build/ (depending on the build system that is being used.

Reply:
To address in new PR :).